.github/workflows/download-datasets-daily.yml

```
@@ -0,0 +1,51 @@
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11"]
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)